MetadataExtractor/Formats/Jpeg/JpegDnlDirectory.cs

```diff
@@ -0,0 +1,55 @@
+#region License
+//
+// Copyright 2002-2019 Drew Noakes
+// Ported from Java to C# by Yakov Danilov for Imazen LLC in 2014
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+// More information about this project is available at:
+//
+//    https://github.com/drewnoakes/metadata-extractor-dotnet
+//    https://drewnoakes.com/code/exif/
+//
+#endregion
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace MetadataExtractor.Formats.Jpeg
+{
+    /// <summary>Describes tags parsed from JPEG DNL data, holding the image height with information missing from the JPEG SOFx segment</summary>
+    /// <author>Kevin Mott https://github.com/kwhopper</author>
+    [SuppressMessage("ReSharper", "MemberCanBePrivate.Global")]
+    public class JpegDnlDirectory : Directory
+    {
+        /// <summary>The image's height, gleaned from DNL data instead of an SOFx segment</summary>
+        public const int TagImageHeight = 1;
+
+        private static readonly Dictionary<int, string> _tagNameMap = new Dictionary<int, string>
+        {
+            { TagImageHeight, "Image Height" }
+        };
+
+        public JpegDnlDirectory()
+        {
+            SetDescriptor(new JpegDnlDescriptor(this));
+        }
+
+        public override string Name => "JpegDnl";
```

Comment:
Perhaps use name `"JPEG DNL"`. I think that's more consistent with other directory names.